src/test/javascript/spec/component/shared/monaco-editor/monaco-editor.service.spec.ts

```
@@ -0,0 +1,35 @@
+        registerLanguageSpy = jest.spyOn(monaco.languages, 'register');
+        service = TestBed.inject(MonacoEditorService);
+    });
+
+    afterEach(() => {
+        jest.restoreAllMocks();
+    });
+
+    it('should register the custom markdown language', () => {
+        const customMarkdownLanguage = monaco.languages.getLanguages().find((l) => l.id === CUSTOM_MARKDOWN_LANGUAGE_ID);
+        expect(customMarkdownLanguage).toBeDefined();
+        expect(registerLanguageSpy).toHaveBeenCalledExactlyOnceWith({ id: customMarkdownLanguage!.id });
```

Biome

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)